packages/Webkul/Admin/src/Resources/views/catalog/categories/edit.blade.php

```diff
@@ -140,6 +140,17 @@
                                         @php echo str_replace($key, 'Image', $message[0]); @endphp
                                     @endforeach
                                 </span>
+
+                                <label>{{ __('admin::app.catalog.categories.category_banner') }}</label>
+                                <image-wrapper button-label="{{ __('admin::app.catalog.products.add-image-btn-title') }}" input-name="category_banner" :multiple="false" :images='"{{ $category->banner_url }}"'></image-wrapper>
+
+                                <span class="control-error" v-if="{!! $errors->has('image.*') !!}">
+                                    @foreach ($errors->get('image.*') as $key => $message)
+                                        @php echo str_replace($key, 'Image', $message[0]); @endphp
+                                    @endforeach
+                                </span>
+
+                                <span class="control-info mt-10">{{ __('admin::app.catalog.categories.image-size') }}</span>   
```

Comment:
@Rachana1005 
Need to mention the correct suggestion size for the banner image. Here you used the suggestion of the Image size.